mllm/backends/ascend/memory/AscendMemoryManager.cpp

```
@@ -0,0 +1,67 @@
+// Licensed under the MIT License.
+
+#include <acl/acl.h>
+
+#include "AscendMemoryManager.hpp"
+#include "mllm/backends/ascend/AscendCommon.hpp"
+#include "mllm/utils/Common.hpp"
+#include "mllm/utils/Log.hpp"
+
+namespace mllm::ascend {
+
+static AscendMemoryManager g_ascendMemoryManager;
```

Clang (14.0.6)

[error] variable 'g_ascendMemoryManager' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables,-warnings-as-errors)